### PR TITLE
Migrated favicon database to the new site.

### PR DESCRIPTION
**Pull request by nikhil25803**
In this PR
 - Migrated the favicon database to the new site.
 - Created `favicons-datbase.yml` file.
 - Created `favicons-database.html` file and included it to the `Favicons_Database.md` file.

Do let me know if any changes required in this PR @kingthorin :) 

Fixes OWASP/www-community#684

### CODE REVIEW

File: pages/Favicons_Database.md
Comment (kingthorin) [inline pages/Favicons_Database.md:1]: Lowercase the file name please, that just makes life easier in the long run.

File: _includes/favicons-database.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            <th nowrap="nowrap">Source</th>
+        </tr>
+        {% for tool in site.data.favicons-database %}
+        <tr>
+            <td>{{ tool.name }}</td>
+            <td>{{ tool.hash }}</td>
+            <td>{{ tool.algorithm }}</td>
+            <td>{{ tool.source }}</td>
+        </tr>
+        {% endfor %}
```
Comment (kingthorin) [inline _includes/favicons-database.html:16]: End the table?

**Review by kingthorin — COMMENTED** [review]
The lead-in/description I the markdown file could also use some revision but I need more time to think about it.

Looks good overall, though I also haven't pulled and tested locally.

File: _includes/favicons-database.html (hunk 1 above)
Comment (nikhil25803) [reply _includes/favicons-database.html]: Sorry, missed it by mistake. Fixed it.

File: pages/Favicons_Database.md
Comment (nikhil25803) [reply pages/Favicons_Database.md:1]: Done sir ✅

Comment (kingthorin) [reply pages/Favicons_Database.md:1]: I think you missed including this.

Comment (nikhil25803) [reply pages/Favicons_Database.md:1]: I updated the file name earlier with `favicons_database.md` but I don't know why it was not reflected in the PR.  I renamed it now to `favicons-database. md`. Hope you are okay with this,

Comment (kingthorin) [reply pages/Favicons_Database.md:1]: Thanks, that's better. I'll suggest some other edits later tonight or tomorrow. It'll just be  some re-wording. Then this should be good to go.

Comment (nikhil25803) [reply pages/Favicons_Database.md:1]: Great :))

File: pages/favicons-database.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: OWASP Favicon Database
+layout: col-sidebar
+author:
+contributors:
```
Comment (kingthorin) [inline pages/favicons-database.md:5]: You could check the history of the wiki page content, and also add yourself.

File: pages/favicons-database.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+tags: favicons, oss
+permalink: /favicons_database
+---
+
+{% include writers.html %}
+
+So, project has started the adventure of getting the statistics of MD5 fingerprints of most usual favicons.ico. We have faced problems how to enumerate http(s) hosts on Internet. Currently, we have recognized two types of http servers which we want to cover. First type is http servers on network devices and appliances and the second type is normal web servers with virtual hosts support. Idea is to have software enumerated via favicon.ico. How to do that? Take hash (in our case MD5) of favicon.ico and compare it against the known database. This project is about the favicon database itself and process in how to get the database of most frequent ones by crawling internet.
```
Comment (kingthorin) [inline pages/favicons-database.md:12]:
```suggestion
This content represents a community attempt to enumerate/document fingerprints of known favicons (favicon.ico) used on the web. We have faced problems how to enumerate http(s) hosts on the Internet. Currently, we have recognized two types of HTTP servers which we want to cover. The first type are HTTP servers on network devices and appliances. The second type is normal web servers with virtual hosts support. A portion of this effort is to have software enumerated via favicon.ico. How to do that? Take hash (MD5, etc) of favicon.ico and compare it against a known database (such as this one).
```

I don't know, it's still not perfect but it's community content and I think "good enough" for now 😉

File: pages/favicons-database.md (hunk 2 above)
Comment (nikhil25803) [reply pages/favicons-database.md]: Updated it, check once.

File: pages/favicons-database.md (hunk 3 above)
Comment (nikhil25803) [reply pages/favicons-database.md]: It does look good for now, we can make changes in future accordingly.

**Review by kingthorin — APPROVED** [review]
Thanks!

**Comment by kingthorin** [comment]
It's live: https://owasp.org/www-community/favicons_database


Now that I've seen it we probably should include source in the data file but not bother displaying it... I'll tweak that in another PR sometime.

**Comment by nikhil25803** [comment]
Thank you @kingthorin for guiding me throughout.
I look forward to making more significant contributions ahead :)

**Comment by kingthorin** [comment]
The include/template has been adjusted.

**Comment by nikhil25803** [comment]
Great, thanks :))

**Pull request merged**